sdk/eventgrid/arm-eventgrid/test/eventgrid_examples.ts

```diff
@@ -27,7 +27,8 @@ const replaceableVariables: Record<string, string> = {
 };
 
 const recorderOptions: RecorderStartOptions = {
-  envSetupForPlayback: replaceableVariables
+  envSetupForPlayback: replaceableVariables,
+  removeCentralSanitizers: ["AZSDK3493"]
```

Comment:
Are there docs on when and what sanitizers people need to remove?

Reply:
I saw in some packages like `storage-blob`, there is a comment on why we don't need the `x-ms-encryption-key-sha256` sanitizer. I'd love to have this guideline too.

Reply:
Scott is working on the docs https://github.com/Azure/azure-sdk-tools/pull/8142/

I've added comments for all the current usages.